README.md

```diff
@@ -120,6 +120,20 @@ mod ffi {
 }
 ```
 
+## Comparison to other Rust Qt bindings
+
+| project |  integrate into C++ codebase | safe Rust | maintained* | binding mechanism |
+|-------- | ---------------------------- | --------- | ----------- | ----------------- |
+| cxx-qt  |  ✔                           | ✔        |  ✔          | [cxx](https://cxx.rs) plus additional code generation to implement QObject subclasses in Rust and bind them to C++ |
+| [qmetaobject](https://github.com/woboq/qmetaobject-rs/) | ✗ | ✔ | ✔ | [cpp](https://github.com/mystor/rust-cpp) macro to write C++ inline in Rust, plus Rust macros to create QObject subclasses from Rust structs |
```

Comment:
`qmetaobject` can sortof integrate with C++ by creating a `QQmlExtensionPlugin`, although I don't think there is a way to directly use the classes created by Rust like CXX-Qt can..

Reply:
Come to think of it, I wouldn't really call that integrating with C++. That's integrating with QML.